src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -68,11 +70,31 @@ $weekday--margin-right: 10px;
     border-radius: $border-radius--xl;
     opacity: 0.8;
 
+    button {
+      appearance: none;
+      -webkit-appearance: none;
+      border: none;
```

Comment:
Why using vendor prefixed styles here? Why setting these things to `none`?

Reply:
It was done by @sabrinaorban . On my iPad it not affects the UI. Removed this style.